### PR TITLE
[DO NOT MERGE YET] Remove "bc-" prefix of barclamp id's

### PR DESCRIPTION
**Pull request by vwallfahrer**

### CODE REVIEW

File: crowbar_framework/Gemfile [hunk 1]
```diff
@@ -20,7 +20,7 @@ source "https://rubygems.org"
 gem "rails", "~> 4.1.9"
 gem "haml-rails", "~> 0.5.3"
 gem "sass-rails", "~> 4.0.5"
-gem "rainbows-rails", "~> 1.0.1"
+gem "puma", "~> 2.13.4"
```
Comment (tboerger) [inline crowbar_framework/Gemfile]: This is not related to this pr, please drop it.

**Comment by tboerger** [comment]
Do we need some kind of migrations for already existing installations? After dropping the puma bit :+1:

**Comment by vuntz** [comment]
@rsalevsky are you fine doing the migration for this in the upgrade process, or should we do it otherwise?

**Comment by rsalevsky** [comment]
@vuntz I'm fine with doing the migration while the upgrade process it's much less work then writing a migration now.

**Comment by tboerger** [comment]
:+1:

**Comment by mjura** [comment]
for me OK too +1

**Comment by vuntz** [comment]
Can you rebase?

**Comment by vwallfahrer** [comment]
Continue in new PR https://github.com/crowbar/crowbar-core/pull/46
